contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -18,6 +18,7 @@ abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHand
   // solhint-disable-next-line var-name-mixedcase
   uint8 public immutable MAX_TOKEN_SHARES;
   mapping(uint80 => StrategyOwnerAndVersion) internal _strategies;
+  mapping(uint80 => address) internal _strategiesPendingOwner;
```

Comment:
Maybe we should make this public? We usually try to make sure that everything can be read off-chain in some form. It doesn't hurt to make it public, and if we need to figure out what is stored, it's easier than trying to read internal/private variables

Also, let's try to add a function to the interface, explaining what this does